templates/Base.dockerfile.j2

```diff
@@ -5,18 +5,15 @@
 #
 # Copyright (c) {{ now.year }}, NVIDIA CORPORATION.
 
-ARG CUDA_VERSION={{ DEFAULT_CUDA_VERSION }}
-ARG CUDA_MAJORMINOR_VERSION=${CUDA_VERSION}
-ARG LINUX_VERSION={{ os }}
-ARG PYTHON_VERSION={{ DEFAULT_PYTHON_VERSION }}
+ARG CUDA_VER={{ DEFAULT_CUDA_VERSION }}
```

Comment:
For now I left the settings file unchanged and these variables `DEFAULT_CUDA_VERSION` and `DEFAULT_PYTHON_VERSION`

Should we change them to `CUDA_VER` and `PYTHON_VER` to match the build args or keep them separate @ajschmidt8?